### PR TITLE
Add multiple language support in the theme

### PR DESCRIPTION
**Pull request by ebetica**
You now have two extra things to do:

- For each language, add its own table of contents in the config. They are effectively their own websites.
- Whenever a webpage has multiple language versions, you have to provide a "lang-ref" and "lang" key in the header, that tells it which pages are ones of the same version, and what language that webpage is in. I found this easier to code than keying on a file path, and allows for page file names in different languages if need be.

Webpages without translations will unfortunately not exist. We can simply copy the english versions over for now I guess...

### CODE REVIEW

File: docs/zh/01-1.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+## [Hierarchical Representation of the Visual Cortex](0:45:27-0:51:40)
+
+Experiments by Fukushima gave us an understanding of how our brain interprets the input to our eyes. In summary, it was discovered that neurons in front of our retina compress the input (known as contrast normalization) and the signal travels from our eyes to our brain. After this, the image gets processed in stages and certain neurons get activated for certain categories. Hence, the visual cortex does pattern recognition in a hierarchical manner.
+
+Experiments were conducted that poked electrodes in specific areas of the visual cortex, specifically the V1 area, and realized that certain neurons react to motifs that appear in a very small area in a visual field and similarly with neighboring neurons and neighboring areas in the visual field. Additionally, neurons that react to the same visual field, react to different types of edges in an organized manner (e.g. vertical or horizontal edges). It is also important to note that there's also the idea that the visual process is essentially a feed forward process. Hence, somehow fast recognition can be done without some recurrent connections.
```
Comment (Atcold) [inline docs/zh/01-1.md]: Please, remove duplicate content.

File: docs/zh/01-2.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+How to learn representations (good features)?
+
+Manifold hypothesis: natural data lives in a low-dimensional manifold. Set of possible images is essentially infinite, set of "natural" images is a tiny subset. For example: for an image of a person, the set of possible images is on the order of magnitude of the number of face muscles they can move (degrees of freedom) ~ 50. An ideal (and unrealistic) feature extractor represents all the factors of variation (each of the muscles, lighting, etc).
+
+Q&A from the end of lecture:
+
+- For the face example, could some other dimensionality reduction technique (i.e. PCA) extract these features?
+  - Answer: would only work if the manifold surface is a hyperplane, which it is not
+
```
Comment (Atcold) [inline docs/zh/01-2.md]: Please, remove duplicate content.

File: docs/zh/01-3.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+
+### Random Neural Network
+
+Lastly, we visualize the transformation performed by a simple, untrained neural network. The network consists of a linear layer, which performs an affine transformation, followed by a tanh non-linearity, and finally another linear layer. Examining the transformation in Fig. 6, we see that it is unlike the linear and non-linear transformations seen earlier. Going forward, we will see how to make these transformations performed by neural networks useful for our end goal of classification.
+
+<center>
+<img src="untrained_nn_transformation_lab1.png" width="200px" /><br>
+Figure 6:  Transformation from an untrained neural network
+</center>
```
Comment (Atcold) [inline docs/zh/01-3.md]: Please, remove duplicate content.

**Comment by ebetica** [comment]
Okay - duplicates removed.

File: docs/_config.yml [hunk 4]
```diff
@@ -61,3 +63,23 @@ chapters:
       - path: chapters/04-1.md
 #epilogues:
 #  - path: chapters/end.md
+
+# The only required component of multiple languages is
+# a `zh.md` for the index, or `zh/index.md`.
+# The only supported config so far is "title" and "tagline"
+zh:
+  title:   'Deep Learning in 中文'
```
Comment (Atcold) [inline docs/_config.yml]: `title:   '深度学习'`

File: docs/_config.yml [hunk 5]
```diff
@@ -61,3 +63,23 @@ chapters:
...
+  tagline: 'Gotta learn them all in 中文'
```
Comment (Atcold) [inline docs/_config.yml]: Something I cannot translate.

Comment (Atcold) [reply docs/_config.yml]: I mean, I suck.

**Pull request merged**